qt/python/mantidqt/widgets/instrumentview/instrument_view.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+
+import functools
+from enum import IntEnum
+from mantidqt.utils.qt.qappthreadcall import QAppThreadCall
+from mantidqt.widgets.instrumentview.presenter import InstrumentViewPresenter
+
+
+def safe_qthread(func):
+    """decorator function to move given call to main QappThread"""
+    @functools.wraps(func)
+    def _wrapped(*args, **kwargs):
+        return QAppThreadCall(func)(*args, **kwargs)
+
+    return _wrapped
+
+
+class SurfaceType(IntEnum):
+    """Enumertor for the surface type in Instrument View"""
+
+    Full3D = 0
+    CylindricalX = 1
+    CylindricalY = 2
+    CylindricalZ = 3
+    SphericalX = 4
+    SphericalY = 5
+    SphericalZ = 6
+    SideBySide = 7
+
+
+class TabName(IntEnum):
+    """Enumerator for the Tab in Instrument View"""
+
+    Render = 0
+    Pick = 1
+    Draw = 2
+    Instrument = 3
+
+
+class pyInstrumentView:
```

Comment:
Should we just call this `InstrumentView`? The `py` part seems a little superfluous, possibly?

Reply:
With the new implementation, there is no longer a dedicated wrapper class anymore.